spalp_example/src/EventSubscriber/SpalpExampleAppIdsAlterSubscriber.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Drupal\spalp_example\EventSubscriber;
+
+use Drupal\spalp\Event\SpalpAppIdsAlterEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Class SpalpExampleAppIdsAlterSubscriber.
+ *
+ * @package Drupal\spalp\EventSubscriber
+ */
+class SpalpExampleAppIdsAlterSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    return [
```

Comment:
try to improve readability of your code. Consider formatting it as shown in the example below:
https://github.com/dakkusingh/search_api_elasticsearch_attachments/blob/8.x-1.x/src/EventSubscriber/BuildSearchParams.php#L19